framework/src/vectorpostprocessors/PointSamplerBase.C

```diff
@@ -144,6 +144,18 @@ PointSamplerBase::finalize()
   SamplerBase::finalize();
 }
 
+void
+PointSamplerBase::setPointsVector(std::vector<Point> points)
```

Comment:
Is there a good reason why we're making a copy here?

Reply:
Nope - but it doesn't really matter, the compiler should optimize one of these copies out. It won't copy twice when it makes another copy during the _points assignment. Note that I created a "transfer" version of this function which allows us to move a vector instead. I can fix this though just to be more clear.